docker-compose.yml

```diff
@@ -19,12 +19,6 @@ services:
       HASURA_GRAPHQL_ENABLE_CONSOLE: "true"
       HASURA_GRAPHQL_ADMIN_SECRET: key
       HASURA_GRAPHQL_JWT_SECRET: '${SKILLZ_GRAPHQL_JWT_SECRET?SKILLZ_GRAPHQL_JWT_SECRET}'
-      HASURA_ACHIEVEMENTS_ENDPOINT: http://app:3000/api/achievement
-  app:
-    build:
-      context: .
-      dockerfile: app.Dockerfile
-    ports:
-      - 3000:3000
+      HASURA_ACHIEVEMENTS_ENDPOINT: http://host.docker.internal:3000/api/achievement
```

Comment:
Very cool! Did not know about this!

Reply:
This on works on docker for windows and mac though, we need to update this so this uses the .env (same as the jwt key), so devs on linux can use their docker bridge IP instead 😄 More info here https://hasura.io/docs/latest/graphql/core/guides/docker-networking.html